lib/rouge/lexers/mojo.rb

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*- #
 # frozen_string_literal: true
 
+require('rouge/lexers/python')
```

Comment:
**suggestion:** Could we use the `load_lexer` utility for better consistency?

```py
module Rouge
  module Lexers
    load_lexer 'python.rb'
   
    class Mojo < Python
       ...
```

Reply:
That sounds better! I changed it to your suggestion.